base/post_hook_rocm_smi_lib.cmake

```diff
@@ -1,6 +1,6 @@
 target_include_directories(rocm_smi64 PUBLIC
-  "$<INSTALL_INTERFACE:lib/rocm_sysdeps/include>"
+  "$<INSTALL_INTERFACE:include>"
```

Comment:
If CI says good, this is fine. This was working around an issue, but fine if no longer needed.

Reply:
As far as I can see, with `rocm_smi.h` beeing placed in `include/rocm_smi/rocm_smi.h`, this is where `INSTALL_INTERFACE` should point to. Without this change the header wasn't found.